chain-api/src/validators/decorators.ts

```diff
@@ -38,6 +38,28 @@ export function IsWholeNumber(property: string, validationOptions?: ValidationOp
   };
 }
 
+export function IsNot(
+  property: string,
+  condition: (instance: Record<string, unknown>) => unknown,
+  validationOptions?: ValidationOptions
+) {
+  return function (object: Record<string, unknown>, propertyName: string): void {
+    registerDecorator({
+      name: "isNot",
+      target: object.constructor,
+      propertyName,
+      constraints: [property],
+      options: validationOptions,
+      validator: {
+        // eslint-disable-next-line
+        validate(value: Record<string, unknown>, args: ValidationArguments) {
+          return true;
```

Comment:
It lacks proper implementation. @herr-bubu please find out if it's really used anywhere.